test/libdnf/advisory/test_advisory_module.cpp

```diff
@@ -82,3 +82,37 @@ void AdvisoryAdvisoryModuleTest::test_get_advisory_collection() {
     size_t target_size = 2;
     CPPUNIT_ASSERT_EQUAL(target_size, out_mods.size());
 }
+
+void AdvisoryAdvisoryModuleTest::test_copy_move() {
```

Comment:
Not saying the tests are useless, but with the new `ImplPtr` class there's little room for issues, right? This is a somewhat non-trivial amount of code and if we wanna test these, then we should add the same tests for all our classes. That'll be an awful lot of additional code which I think is not really worth it?

Reply:
I removed the tests for the copy/move constructors/assignments of the `AdvisoryModule` and` AdvisoryPackage` classes. Instead, I added tests for the `ImplPtr` class.